tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/upgrade/ICodeUpgradleConstants.java

```diff
@@ -0,0 +1,27 @@
+/*******************************************************************************
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ *
+ *******************************************************************************/
+
+package com.liferay.ide.project.core.upgrade;
+
+/**
+ * @author Lovett Li
+ */
+public interface ICodeUpgradleConstants
+{
+    String LAYOUT_UPGRADE_TO_LIFERAY_PLUGIN_SDK_7 = "Upgrade to liferay plugin sdk 7";
+    String PROJECT_NAME_PLUGIN_SDK_7 = "plugins.sdk-7.0";
```

Comment:
This name we just give a default project name when user open the upgrade view and select upgrade to liferay plugin sdk. We can change it to any value.